### PR TITLE
Lh theme modal

### PR DESCRIPTION
**Pull request by lancehamilton24**
- finalized modal for theme customization.

### CODE REVIEW

**Comment by MCrank** [comment]
@lancehamilton24 Hey there are still some merge conflict <<<<< HEAD and >>>>>> master tags in the code preventing it from running.

**Comment by MCrank** [comment]
- Unchecking the Large Text option does not revert the text size

File: javascripts/events/navbarEvents.js [hunk 1]
```diff
@@ -43,4 +43,31 @@ const clearMsgEvent = () => {
...
 };
 
-export { navbarEnterEvents, clearMsgEvent, navBarUserSelectEvents };
+const textSize = () => {
+  const largeText = document.getElementById("large-text");
+  largeText.addEventListener("change", e => {
+    if (largeText.checked) {
+      document.getElementById("message-output").style.fontSize = "2rem";
+    } else {
+      document.getElementById("message-output").style.fonSize = "1rem";
```
Comment (MCrank) [inline javascripts/events/navbarEvents.js]: Missing the `t` on fontSize

**Comment by m-khezri** [comment]
Lance, It's great - Good job!
Recommendations:
* I think texts look huge. I recommend to use h5 for title and p tag for descriptions.
* Add 'fade' in modal class to add fade effect

**Comment by mjleavell** [comment]
@lancehamilton24 this is sick! Great job! I agree with @m-khezri in possible reducing the size of the text in the modal. Otherwise, it is AWESOME 👍 👍 👍

**Comment by mjleavell** [comment]
and forgot to add that yes, unchecking the "large text" checkbox doesnt change the text back to the original size

**Comment by MCrank** [comment]
>           and forgot to add that yes, unchecking the "large text" checkbox doesnt change the text back to the original size

@mjleavell I think Lance just pushed a fix for that.  Try and Pull latest

**Comment by lancehamilton24** [comment]
Yeah I just caught the message. I'll change the tags and add a fade.

**Comment by mjleavell** [comment]
> > ```
> >       and forgot to add that yes, unchecking the "large text" checkbox doesnt change the text back to the original size
> > ```
> 
> @mjleavell I think Lance just pushed a fix for that. Try and Pull latest

Yes, he did. I was about to comment that my branch hadnt been updated...,my bad. it's working now, but i get this error in the console 
image

**Comment by MCrank** [comment]
Man this is super sweet! 👍 👍 👍 👍 👍

**Comment by MCrank** [comment]
💯 👍 Works like a champ now and the button confusion is fixed

**Comment by mjleavell** [comment]
👍 👍 awesome!!

**Pull request merged**